pkg/deploy/controller/deployment/controller_test.go

```diff
@@ -240,6 +241,16 @@ func TestHandle_deployerPodAlreadyExists(t *testing.T) {
 
 	err := controller.Handle(deployment)
 
+	var updatedDeployment *kapi.ReplicationController
```

Comment:
Would it be less work to just assign `updatedDeployment` right from the `ReactFn` for the `update-replicationController` case?